src/zcl_abapgit_repo_srv.clas.abap

```diff
@@ -291,7 +291,6 @@ CLASS ZCL_ABAPGIT_REPO_SRV IMPLEMENTATION.
 
     add( ro_repo ).
 
-    ro_repo->refresh( ).
     ro_repo->find_remote_dot_abapgit( ).
```

Comment:
I guess this call will fail?

Reply:
After deletion of ro_repo->refresh( ) the creation of repos with a existing git URL seems to work furthermore.
Have I overlooked something?

Reply:
if find_remote_dot_abapgit() does not find the file(which it will not since mt_remote is not filled), then it will use the default .abapgit.xml settings, which is probably what you have tested against?

Reply:
you're right. It was default .abapgit.xml.
Maybe TRY/CATCH is the better solution?